proto/osmosis/superfluid/tx.proto

```diff
@@ -98,6 +102,21 @@ message MsgLockAndSuperfluidDelegate {
 }
 message MsgLockAndSuperfluidDelegateResponse { uint64 ID = 1; }
 
+// MsgCreateFullRangePositionAndSuperfluidDelegate creates a full range position
+// in a concentrated liquidity pool, then superfluid delegtes.
+message MsgCreateFullRangePositionAndSuperfluidDelegate {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  repeated cosmos.base.v1beta1.Coin coins = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+  string val_addr = 3;
+  uint64 pool_id = 4 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+}
+message MsgCreateFullRangePositionAndSuperfluidDelegateResponse {
+  uint64 ID = 1;
```

Comment:
Can we make this more descriptive? I just started reviewing and am currently unsure if this is the position ID or the lock ID. I am sure this will become clearer later though. Also, shouldn't this return both lock and position ID?